queries/thrift/textobjects.scm

```diff
@@ -0,0 +1,130 @@
+; Assignments
+
+(namespace_uri
+  (uri_def) @assignment.lhs
+  "="
+  (uri) @assignment.inner @assignment.rhs) @assignment.outer
+
+(const
+  (
+    "const"
+    (_) @assignment.lhs
+    "="
+    (const_value) @assignment.inner @assignment.rhs
+  ) @assignment.outer)
+
+(enum
+  (
+    (enum_member) @assignment.lhs
+    "="
+    (number) @assignment.inner @assignment.rhs
+  ) @assignment.outer)
+
+(field
+  (
+    (_) @assignment.lhs
+    "="
+    (const_value) @assignment.inner @assignment.rhs
+  ) @assignment.outer)
+
+(function_parameter
+  (fb_annotation)? ; don't catch this under @assignment.lhs
+  (
+    (_) @assignment.lhs
+    "="
+    (const_value) @assignment.inner @assignment.rhs
+  ) @assignment.outer) 
+
+(exception_parameter
+  (
+    (_) @assignment.lhs
+    "="
+    (const_value) @assignment.inner @assignment.rhs
+  ) @assignment.outer)
+
+(annotation
+  (
+    (annotation_definition) @assignment.lhs
+    "="
+    (annotation_value) @assignment.inner @assignment.rhs
+  ) @assignment.outer)
+
+((field_identifier) @assignment.lhs
+  "="
+  (const_value) @assignment.inner @assignment.rhs) @assignment.outer
+
+; Attributes
+
+(xsd_attrs) @attribute.outer
+(xsd_attrs
+  "{" (_) @attribute.inner "}")
+
+(annotation) @attribute.outer
+(annotation
+  "(" (_) @attribute.inner ")")
+
+(fb_annotation) @attribute.outer
+(fb_annotation
+  (fb_annotation_definition
+    "{" (_) @attribute.inner "}"))
+
+; Blocks
+
+("{" (_) @block.inner "}") @block.outer
+
+; Classes
+
+(enum) @class.outer
+(enum
+    "{" (_) @class.inner "}")
+
+(senum) @class.outer
+(senum
+    "{" (_) @class.inner "}")
+
+(struct) @class.outer
+(struct
+    "{" (_) @class.inner "}")
+
+(union) @class.outer
+(union
+    "{" (_) @class.inner "}")
+
+(exception) @class.outer
+(exception
+    "{" (_) @class.inner "}")
```

Comment:
this is guaranteed to be only one node?

Reply:
It's a repeat of "field"

Reply:
![image](https://user-images.githubusercontent.com/29718261/222559549-98f1c25c-6889-4509-984a-3f7a43136931.png)

Reply:
so it could be multiple nodes and this query wouldn't work then